traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/util/Fetcher.java

```diff
@@ -233,4 +239,23 @@ public void createStringBuilderFromResponse (final StringBuilder sb, final HttpU
 			}
 		}
 	}
+
+	private void connectionFailed(final HttpURLConnection http, final String url) {
+		String httpUrl = url;
+		String responseCode = "(none)";
+		try {
+			httpUrl = http.getURL().toString();
+			responseCode = String.valueOf(http.getResponseCode());
+		} catch (Exception e2) {
+			// Don't care
+			LOGGER.info("Exception during call attempt to retrieve url or responseCode from http");
+		}
+
+		LOGGER.error("Failed Http Request to " + httpUrl + responseCode);
```

Comment:
Can we change this to `LOGGER.error("Failed HTTP request to " + httpUrl + " status code: " + responseCode);`?

Reply:
Yeah - duh and whoops ... definitely